fabric-fluid-v1/src/main/java/net/fabricmc/fabric/api/fluid/v1/math/Drops.java

```diff
@@ -0,0 +1,74 @@
+package net.fabricmc.fabric.api.fluid.v1.math;
+
+import net.minecraft.nbt.CompoundTag;
+
+/**
+ * Drops are the standard unit of fluid for the fluid api.
+ * 1/{@link #BUCKET} is the smallest amount of fluid that can be described with the Drops system.
+ * Stored in a long, at one bucket per tick, would take around 11.6 million years to overflow!
+ */
+public final class Drops {
+	/**
+	 * To avoid constant inlining use {@link #getBucket()} and {@link #getBuckets(long)}.
+	 * this number is divisible by all numbers from 1 to 10 inclusive.
+	 */
+	private static final long BUCKET = 2520L;
```

Comment:
I've said my piece on this; the best option for commonly-used denominators is 3240. 7 just isn't worth it.

Reply:
3240 is indeed better than 2520 in a way that it can be divided by 81.